pkg/registry/apps/playlist/conversions.go

```diff
@@ -135,12 +129,5 @@ func getLegacyID(item *unstructured.Unstructured) int64 {
 	if err != nil {
 		return 0
 	}
-	info, _ := meta.GetRepositoryInfo()
-	if info != nil && info.Name == "SQL" {
-		i, err := strconv.ParseInt(info.Path, 10, 64)
-		if err == nil {
-			return i
-		}
-	}
-	return 0
+	return meta.GetDeprecatedInternalID() // nolint:staticcheck
```

Comment:
This returns 0 if not set if someone is wondering during review. https://github.com/grafana/grafana/blob/d91d7297492286b9ac61cbf2e6232cedde0579f5/pkg/apimachinery/utils/meta.go#L309-L323